.gitlab/deploy_packages/e2e.yml

```
@@ -1,4 +1,6 @@
-# Jobs that deploy agent packages on QA environment, to be used by e2e tests
+# Jobs that deploy agent packages on QA environment, to be used by e2e tests.
+# We use two separate jobs for Windows and Linux so that a failure in deploying the
+# Linux / Windows scrip doesn't impact the other OS (i.e. Windows scripts failing to be signed blocking Linux E2E tests).
 
 qa_installer_script:
```

Should we call this one like this?
```suggestion
qa_installer_script_linux:
```